Doc/whatsnew/3.13.rst

```diff
@@ -97,6 +97,15 @@ pathlib
   :meth:`~pathlib.Path.rglob`.
   (Contributed by Barney Gale in :gh:`77609`.)
 
+threading
+---------
+
+* Passing any arguments to :func:`threading.RLock` is now restricted.
+  Previously C version allowed any numbers of args and kwargs,
+  but they were just ignored. Python version never allowed any arguments.
```

Comment:
This statement isn't accurate; Python <= 2.7 and 3.1 accepted a single undocumented `verbose` argument.